### ISSUE TITLE
Close issues that refer to Godot Engine 3 but don't exist in Godot Engine 4

### ISSUE THREAD

**Issue by fire** [Geequlim/ECMAScript#173]
Thoughts? Comments?

**Comment by nmerget** [comment]
Yes, this is a good idea. Currently, I want to go through all issues to see which are solved and which ones need labels to categorize them. I saw that a lot of issues are already answered, those "FAQs" should move to the documentation. 
If you know which issues can be closed besides this, we should close them, or at least ask the people that we want to close it (or maybe add a bot to auto-close them)
